kalite/control_panel/tests/control_panel.py

```diff
@@ -551,60 +557,60 @@ def test_device_log_csv_endpoint(self):
         self.assertEqual(len(rows), 2, "API response incorrect")
 
 
-class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
+# class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
```

Comment:
> I get what you're saying, my concern is that the tests won't be re-implemented

That was spot on :)

I'm gonna get a status update on this, if it still fails, I remove the stale code.